src/components/Unknown/Switcher/styles.ts

```diff
@@ -0,0 +1,27 @@
+import { Theme } from '@mui/material';
+import { createStyles, makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    switcherButton: {
+      minWidth: 100,
+      borderRadius: `${theme.shape.borderRadius}px !important`,

```

Comment:
why !important?

Reply:
this is the only way to override default border-radius of ToggleButton